source/plugin/entities/spawning.rst

```diff
@@ -35,8 +32,7 @@ For example, let's try to spawn a Creeper:
         World world = spawnLocation.getExtent();
         Entity creeper = world
             .createEntity(EntityTypes.CREEPER, spawnLocation.getPosition());
-        SpawnCause spawnCause = SpawnCause.builder().type(SpawnTypes.PLUGIN).build();
-        world.spawnEntity(creeper, Cause.source(spawnCause).build());
+        world.spawnEntity(creeper);
```

Comment:
Should be pushing to the cause stack frame for a SpawnType.

Reply:
Fixed